packages/frontend/src/components/accounts/batch_import_accounts/index.js

```diff
@@ -274,7 +275,11 @@ const BatchImportAccounts = ({ onCancel }) => {
     const [accountsData, setAccountsData] = useState(null);
 
     const handlePublicKey = useCallback((publicKey) => {
-        setAccountsData(decodeAccountsFrom(location.hash, publicKey));
+        const accounts = decryptAccountData({
+            ciphertext: location.hash?.substring(1),
+            secretKey: publicKey,
+        });
+        setAccountsData(accounts.map(({ accountId, privateKey }) => [accountId, privateKey, null]));
```

Comment:
Do you know what the `null` would have been in the original implementation? I didn't think we ever passed anything else in besides account ID and key...

Reply:
existing implementation did pass third argument which was ledger key path. (Also if the account wasn't ledger, it actually passed null as well)